hello_world/blockchain_app/hello_module/hello_module.js

```diff
@@ -0,0 +1,67 @@
+const { BaseModule, codec } = require('lisk-sdk');
+const { HelloAsset, HelloAssetID } = require('./hello_asset');
+const {
+    helloCounterSchema,
+    helloAssetSchema,
+    CHAIN_STATE_HELLO_COUNTER
+} = require('./schemas');
+
+class HelloModule extends BaseModule {
+    name = 'hello';
+    id = 1000;
+    accountSchema = {
+        type: 'object',
+        properties: {
+            helloMessage: {
+                fieldNumber: 1,
+                dataType: 'string',
+            },
+        },
+        default: {
+            helloMessage: '',
+        },
+    };
+    transactionAssets = [ new HelloAsset() ];
+    actions = {
+        amountOfHellos: async () => {
+            const res = await this._dataAccess.getChainState(CHAIN_STATE_HELLO_COUNTER);
+            const count = codec.decode(
+                helloCounterSchema,
+                res
+            );
```

Comment:
What if someone call this action before any transaction sent. Would be nice if you add `afterGenesisBlockAppy` hook and set this counter to zero.

Reply:
Done